.travis.yml

```diff
@@ -1,14 +1,14 @@
-language: node_js
-node_js:
-- "4"
-- "node"
 sudo: false
+language: node_js
 cache:
   directories:
-  - node_modules
-install:
-- npm install
-- npm update
+    - node_modules
+node_js:
+  - "4"
+  - "node"
+before_install:
+  - npm update
+  - npm prune
```

Comment:
Ah - I was specifically trying to ensure that the `npm update` happens before the build step, so if you're right about the `prepublish` change then that sounds ideal. I'll give it a shot.